stanovy.tex

```diff
@@ -159,23 +171,24 @@ \section{Členská schůze}
         \item hodnotit činnost dalších orgánů Spolku i jejich členů,
         \item rozhodnout o dobrovolném rozpuštění Spolku,
         \item rozhodnout o udělení řádu \textit{Přítel severské zvěře} navrženého výborem Spolku,
-        \item rozhodnout o výhodách aktivních členů Spolku navrhovaných výborem Spolku. 
+        \item rozhodnout o výhodách aktivních a emeritních členů Spolku navrhovaných výborem Spolku. 
     \end{enumerate}
     \item Členská schůze je svolávána výborem Spolku podle potřeby, nejméně 
-    však jednou za šest měsíců.
-    \item Na žádost nejméně jedné třetiny členů musí být svolána mimořádná 
+    však jednou za dvanáct měsíců.
+    \item Na žádost nejméně jedné třetiny členů nebo alespoň 10 aktivních členů, podle toho, která hodnota je menší, musí být svolána mimořádná 
```

Comment:
Logika je taková, že když je nás hodně tak by mělo stačit nějaké menší množství než shánět tu třetinu. Když by členů bylo málo, pak ta třetina je dostačující.